cime/src/share/util/shr_spfn_mod.F90

```diff
@@ -11,7 +11,7 @@
 ! Intel also has them (and Cray), but as of mid-2015, our implementation is
 ! actually faster, in part because they do not properly vectorize, so we
 ! pretend that the compiler version doesn't exist.
-#if defined CPRIBM || defined __GFORTRAN__
+#if defined CPRIBM || defined __GFORTRAN__ || defined CPRINTEL
```

Comment:
To confine the changes to E3SM, this update needs to be reverted (this activates intrinsics for all intel compilers).

Reply:
Yes I'm doing that.